tools/gn

```diff
@@ -152,6 +152,8 @@ def to_gn_args(args):
         gn_args['dart_runtime_mode'] = 'profile'
     elif runtime_mode == 'dynamic_release':
         gn_args['dart_runtime_mode'] = 'release'
+    elif runtime_mode == 'jit_release':
+        gn_args['dart_runtime_mode'] = 'release';
```

Comment:
`jit_release` so both variants can in the out directory and engineers tinker on builds locally without one clobbering another.

Reply:
dart_runtime_mode is used by the Dart SDK. Otherwise, the output directory I have locally is host_jit_release(_unopt) and android_jit_release(_unopt) so this should be okay?

Reply:
My bad. I misread this line to mean it was the out directory. Thanks for the clarification.